operator-common/src/main/java/io/strimzi/operator/cluster/operator/resource/cruisecontrol/CruiseControlRebalanceKeys.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.cluster.operator.resource.cruisecontrol;
+
+/**
+ * This enum contains the keys used in the various levels of the JSON object returned by the Cruise Control
+ * rebalance endpoint.
+ */
+public enum CruiseControlRebalanceKeys {
```

Comment:
```suggestion
public enum CruiseControlRebalance {
```
Might be a fine name, as it is our interpretation of their JSON Object, if possible any parsing of this object we should move into this class, i.e. https://github.com/strimzi/strimzi-kafka-operator/pull/3471/files#diff-bd35cca8a08a6a8f6019797f7532096fR518-R525

As right now the understanding of this objects construction is spread out across several files.

We could write new methods such as how to access before/after brokers fields or even marshal the JSONObject into a pojo for cleaner interactions.

Reply:
This is a very good point. It would be good to have a single source of truth for the rebalance JSON. Do you think it is worth doing that work as part of this PR or a separate one?

Reply:
I think either, it might be clearer in a followup PR though, should be in theory quite simple to move all the logic, just look for references to JSONObjects, also will mean the test code is a lot cleaner.

Reply:
Thanks @samuel-hawker, I will clean this up in a followup PR.